libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -49,6 +50,14 @@ @implementation SFRestAPI
 @synthesize activeRequests=_activeRequests;
 @synthesize currentNetwork=_currentNetwork;
 
+__strong static NSDateFormatter *httpDateFormatter;
+
++ (void) initialize {
```

Comment:
It's good practice to wrap the implementation of an `initialize` method in a conditional like `if (self == [SFRestAPI class]) { }`, because this method will be called once for each `SFRestAPI` class/subclass occurrence.